frontend/src/components/BannersWrapper.vue

```diff
@@ -19,6 +19,7 @@
   <template v-if="shouldShowExternalUrlBanner">
     <BannerExternalUrl />
   </template>
+  <BannerAnnouncement />
```

Comment:
This should be guarded in the similar way as other banners

Reply:
Keep it as is per offline discussion